src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/FsCrawlerImplAllParametersIT.java

```diff
@@ -525,6 +525,24 @@ public void test_filename_as_id() throws Exception {
         }), equalTo(true));
     }
 
+    public void test_add_as_inner_object() throws Exception {
```

Comment:
I think there is something wrong with the implementation.
You are setting here `add_as_inner_object` to `true` but at the end the json document is not copied to the `object` field.